api/src/app.ts

```diff
@@ -13,20 +13,24 @@ import {
 } from './middleware/critterbase-proxy';
 import { rootAPIDoc } from './openapi/root-api-doc';
 import { authenticateRequest, authenticateRequestOptional } from './request-handlers/security/authentication';
+import { loadEvironmentVariables } from './utils/env-config';
 import { scanFileForVirus } from './utils/file-utils';
 import { getLogger } from './utils/logger';
 
 const defaultLog = getLogger('app');
 
+// Load and validate the environment variables
+loadEvironmentVariables();
```

Comment:
Does this need to be called before calling `getLogger()` above?

Reply:
I don't think it "needs" to, but calling it higher up is probably preferred. It will crash regardless if the .env is not to spec.

Reply:
Right, ya I guess the process.env values still exist, they just havent been verified yet